client/marklogicClient.ts

```diff
@@ -73,6 +73,12 @@ export class MlClientParameters {
     ssl: boolean;
     pathToCa: string;
     rejectUnauthorized: boolean;
+    useProxy: boolean;
+    basePath: string;
+    managePath: string;
```

Comment:
Include "BasePath" as the suffix for these - that's consistent with ml-gradle and avoids any confusion over what kind of "path" it is.